src/Box.js

```diff
@@ -4,11 +4,38 @@ import PropTypes from 'prop-types'
 
 const align = responsiveStyle('text-align', 'align')
 
+
+function boxShadow(props) {
+  const shadowColor = 'rgba(0,0,0,0.08)'
+  const shadows = [
+    `0 0 4px 0 ${shadowColor}`, 
+    `0 8px 8px 0 ${shadowColor}`, 
+    `0 16px 16px 0 ${shadowColor}`, 
+    `0 32px 32px 0 ${shadowColor}`
```

Comment:
I think the 0, 4, 8, 16, 32 px sizes should come from the theme probably

Reply:
@odmln

Reply:
I think it's okay to not pull from the `space` scale, but I *do* think these should be added to the `theme.js` file as a constant